subsys/mgmt/fmfu/src/fmfu_mgmt.c

```diff
@@ -267,16 +278,13 @@ static struct mgmt_group fmfu_mgmt_group = {
 	.mg_group_id = (MGMT_GROUP_ID_PERUSER + 1),
 };
 
-int fmfu_mgmt_init(void)
+int fmfu_mgmt_register_group(fmfu_callback_t client_callback)
 {
-	struct nrf_modem_full_dfu_digest digest;
-
-	int err = nrf_modem_full_dfu_init(&digest);
-
-	if (err != 0) {
-		LOG_ERR("nrf_fmfu_init failed, errno: %d\n.", errno);
-		return err;
+	if (client_callback == NULL) {
+		return -EINVAL;
 	}
+
+	callback = client_callback;
 	mgmt_register_group(&fmfu_mgmt_group);
```

Comment:
No return value from this function? (I know its not part of this PR, but we could fix it now since we are visiting :D )

Reply:
mgmt_register_group does not return any value 
https://github.com/nrfconnect/sdk-mcumgr/blob/b18c7b8e3f1fbfc99e0f02102a9d8e3b60304959/mgmt/include/mgmt/mgmt.h#L367